index.js

```diff
@@ -127,4 +123,34 @@ projectBtn.forEach((button) => {
       }
     }
   });
+});
+
+// form validation
+// close popup  if pressed anywhere
+document.onclick = (e) => {
+  if (e.target.id !== 'id-menu-bar') {
+    menu.classList.remove('active');
+    logo.classList.remove('active');
+    menuButton.classList.remove('active');
+  } else if (
+    e.target.id === 'email'
+    || e.target.id === 'fullname'
+    || e.target.id === 'msg'
+  ) {
+    alert.classList.add('hidden');
+  }
+};
+
+// error message
+function alertMsg(msg) {
+  alert.classList.remove('hidden');
+  alert.textContent = msg;
+}
+
+form.addEventListener('submit', (e) => {
+  const lowerCaseReg = /[a - z]/;
+  if (lowerCaseReg.test(inputEm.value) === false) {
+    alertMsg('Please use small letters for email address');
+    e.preventDefault();
+  }
```

Comment:
- [ ] Great work working on the validation, however, it doesn't seem to work as expected. At the moment the form still submits when uppercase letter is in the email. kindly fix it.

Reply:
Thanks. Changed function